src/mc-schema-broker.c

```diff
@@ -93,6 +93,11 @@ bool mc_schema_broker_request(mc_schema_broker_t *sb, const char *db, const char
     return true;
 }
 
+bool mc_schema_broker_has_multiple_requests(const mc_schema_broker_t *sb) {
+    BSON_ASSERT_PARAM(sb);
+    return sb->ll_len > 1;
```

Comment:
Suggest adding assertions of `mc_schema_broker_has_multiple_requests` in test cases for `mc_schema_broker_request` which ensure adding a request for the same collection (duplicates) preserves the "unique list of collections" invariant which this function requires for correctness.